src/test/java/edu/usf/cutr/pelias/SearchTest.java

```diff
@@ -193,5 +193,21 @@ public void testRequestParameters() throws IOException {
                         "&boundary.rect.min_lat=27.959868&boundary.rect.min_lon=-82.515286" +
                         "&boundary.rect.max_lat=28.131471&boundary.rect.max_lon=-82.367646",
                 request.getUrl().toString());
+
+        request = new SearchRequest.Builder(API_KEY, "burger king")
+                .setApiEndpoint(SEARCH_WITH_FOCUS_ENDPOINT)
+                .setFocusPoint(28.061062d, -82.4132d)
+                .setSources("osm")
+                .setSize(35)
+                .setBoundaryRect(27.959868, -82.515286, 28.131471, -82.367646)
+                .build();
+
+        assertEquals("https://raw.githubusercontent.com/CUTR-at-USF/pelias-client-library/master/src/test/resources/" +
+                        "search-with-focus.json?text=burger+king&api_key=dummyApiKey&sources=osm&size=35" +
```

Comment:
I wrestled with that myself, too.   I think " " can be represented both ways in practice -- if you send either one to the pelias API directly it will interpret them both as a space.  It looks like the difference might be whether it is encoding the "file" part of a URL vs. the "query" part (%20 vs +) and Google is encoding an example query with +.

https://docs.oracle.com/javase/7/docs/api/java/net/URLEncoder.html
says that " " is replaced with "+", though.

And changing the test indeed fails:

```
Failed tests:   testRequestParameters(edu.usf.cutr.pelias.SearchTest): expected:
<...cus.json?text=burger[%20]king&api_key=dummyAp...> 
but was:
<...cus.json?text=burger[+]king&api_key=dummyAp...>
```